types/koa-bodyparser/index.d.ts

```diff
@@ -18,8 +18,8 @@ import * as Koa from "koa";
 
 declare module "koa" {
     interface Request {
-        body: {} | null | undefined;
-        rawBody: {} | null | undefined;
+        body: { [x: string]: any } | null | undefined;
```

Comment:
This change ends up producing a lot of type insecurity, and should probably be avoided. To accomplish this, consider using a typeguard or a runtime type validation library (or, simply cast `body` as an `any` to bypass). For more discussion, see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26330